src/components/landing-tree-stats/LandingTreeStats.tsx

```diff
@@ -14,33 +50,76 @@ const LandingTreeStats: React.FC<LandingTreeStatsProps> = ({
   rainWater,
   carbonEmissions,
 }) => {
-  return (
-    <div className="tree-stats-container">
-      <Typography.Title level={3}>Current Status of our Trees</Typography.Title>
-
-      <div className="map-cards-container">
-        <div className="map-card">
-          <InfoCard header="Money Saved" body={getMoneyString(moneySaved)} />
-        </div>
-
-        <div className="map-card">
-          <InfoCard
-            header="Rain Water Caught"
-            body={`${rainWater.toLocaleString()} gallons`}
-          />
-        </div>
-
-        <div className="map-card">
-          <InfoCard header="Carbon Emissions" body={`${carbonEmissions}%`} />
-        </div>
-      </div>
-
-      <Typography.Paragraph>
-        Learn more about how we got these numbers{' '}
-        <Typography.Link underline>here</Typography.Link>.
-      </Typography.Paragraph>
-    </div>
-  );
+  const { windowType } = useWindowDimensions();
+
+  switch (windowType) {
+    case WindowTypes.Mobile:
+      return (
+        <TreeStatsContainer>
+          <MobileTitle>Current Status of our Trees</MobileTitle>
+          <MobileMapCardsContainer>
+            <RightMargin>
+              <MobileInfoCard
+                header="Money Saved"
+                body={getMoneyString(moneySaved)}
+              />
+            </RightMargin>
+
+            <RightMargin>
+              <MobileInfoCard
+                header="Rain Water Caught"
+                body={`${rainWater.toLocaleString()} gallons`}
+              />
+            </RightMargin>
+
+            <MobileInfoCard
+              header="Carbon Emissions"
+              body={`${carbonEmissions}%`}
+            />
+          </MobileMapCardsContainer>
+
+          <GreyParagraph>
+            Learn more about how we got these numbers{' '}
+            <Link underline>here</Link>.
+          </GreyParagraph>
+        </TreeStatsContainer>
+      );
+
+    default:
```

Comment:
Rather than relying upon the default case, you should explicitly show the desktop case as well. A potential problem this could bring is that if you add a new type to the enum, it will assume this default behavior, even if that is not logically wanted.

Reply:
👍 The default case is used like this in other components too where the desktop, tablet, and narrow desktop share the same styling - I can open another PR to fix that later. Since the default case is required, should we have that display an error message - something like "This browser type is not supported"  (as the types are currently defined, this would never render)?

Reply:
When different cases share the same functionality, you can group them together as follows: 
```
switch (widnowType)
  case WindowTypes.Mobile: 
    // code
    break;
  case WindowTypes.Desktop:
  case WindowTypes.NarrowDesktop:
    // code for both 
    break;  
```

As for what the default case should return, an error message would be good, I think it'd also be acceptable to make  it return a React Fragment, which is basically like returning null JSX `<></>`.